src/compiler/factory/nodeFactory.ts

```diff
@@ -3776,6 +3778,17 @@ namespace ts {
                 : node;
         }
 
+        // @api
+        function createImportCall(importKeyword: Token<SyntaxKind.ImportKeyword> | undefined, typeArguments: readonly TypeNode[] | undefined, argumentsArray: readonly Expression[]) {
+            importKeyword = importKeyword || createToken(SyntaxKind.ImportKeyword);
+            return <ImportCall>createCallExpression(<ImportExpression>importKeyword, typeArguments, argumentsArray);
+        }
+
+        // @api
+        function updateImportCall(node: ImportCall, importKeyword: ImportExpression, typeArguments: readonly TypeNode[] | undefined, argumentsArray: readonly Expression[]): ImportCall {

```

Comment:
We generally don't do `update` methods for compound/custom nodes, only nodes with a specific `SyntaxKind` (with the noted exception being for `updatePropertyAccessChain`, `updateElementAccessChain`, `updateCallChain`, and `updateNonNullExpressionChain`). I'm not sure its necessary given you can just use `updateCallExpression`.